src/libstore/store-api.hh

```diff
@@ -552,7 +552,7 @@ void removeTempRoots();
 ref<Store> openStore(const std::string & uri = getEnv("NIX_REMOTE"));
 
 
-void copyPaths(ref<Store> from, ref<Store> to, const Paths & storePaths);
+void copyPaths(ref<Store> from, ref<Store> to, const Paths & storePaths, bool substitute = false);
```

Comment:
Thanks for keeping this in mind @shlevy ! I'm really looking forward to having #937 in `master`.